sympy/combinatorics/fp_groups.py

```diff
@@ -1113,6 +1113,8 @@ def _simplification_technique_1(rels):
                 rel = rel.eliminate_word(g**(-max_exp), g**(-(max_exp-exp)), _all = True)
         rels[i] = rel
     rels = [r.identity_cyclic_reduction() for r in rels]
+    # include one_syllable_words that are not yet in the list of relators
+    rels += [word for word in one_syllables_words if word not in rels]
```

Comment:
i think you can optmize that using the method extend for lists

Reply:
I'm not sure how that's any different from using `+=`. Could you elaborate?